### PR TITLE
refactor(core): prepublish for user onboarding

### PR DESCRIPTION
**Pull request by yeshamavani**
gh-00

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -80,7 +80,8 @@
     "typescript": "~4.2.4"
   },
   "publishConfig": {
-    "registry": "https://registry.npmjs.org/",
+    "registry": "http://localhost:4873/",
```
Comment (akshatdubeysf) [inline packages/core/package.json:83]: Registry should not be localhost?

File: sandbox/audit-ms-example/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,46 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## <small>0.2.7 (2021-10-13)</small>
```
Comment (akshatdubeysf) [inline sandbox/audit-ms-example/CHANGELOG.md:6]: Why is audit-ms-example effected?

File: sandbox/audit-ms-example/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sourceloop/audit-ms-example",
-  "version": "0.2.2",
+  "version": "0.2.7",
```
Comment (akshatdubeysf) [inline sandbox/audit-ms-example/package.json:3]: ??

**Review by akshatdubeysf — CHANGES_REQUESTED** [review]

**Comment by yeshamavani** [comment]
raised another Pr and merged the same.

**Pull request closed without merging**